Directory.Build.props

```
@@ -25,6 +25,8 @@
     <RunSettingsFilePath>$(RepositoryEngineeringDir).runsettings</RunSettingsFilePath>
     <!-- We don't want to use the workload for AppHost projects in this repo -->
     <SkipAspireWorkloadManifest>true</SkipAspireWorkloadManifest>
+
+    <DashboardPublishedArtifactsOutputPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'DashboardArtifacts', '$(Configuration)'))</DashboardPublishedArtifactsOutputPath>
```

nit: `Path` -> `Dir` - to consistently follow a pattern of using `*Dir` for directories and `*Path` for files.
```suggestion
    <DashboardPublishedArtifactsOutputDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'DashboardArtifacts', '$(Configuration)'))</DashboardPublishedArtifactsOutputDir>
```